README.md

```diff
@@ -203,7 +203,7 @@ Prettier does nothing to help with those kind of rules. They are also the most i
 Install:
 
 ```
-yarn add prettier --dev
+yarn add prettier --dev -E
```

Comment:
Use [`--exact`](https://yarnpkg.com/en/docs/cli/add#toc-yarn-add-exact-e) for consistency?

Reply:
Hah I had `--exact` and switched it to `-E` because it's consistent across yarn and npm. I can switch it back.

Reply:
I'd recommend to use `-DE` if you prefer consistent across yarn and npm.

Reply:
Yeah that's what I'd use bit it's a bit too cryptic for documentation